HLSMergerPlugin/src/main/java/io/antmedia/rest/RestService.java

```diff
@@ -0,0 +1,85 @@
+package io.antmedia.rest;
+
+import com.google.gson.Gson;
+import io.antmedia.plugin.HLSMergerPlugin;
+import io.swagger.v3.oas.annotations.Parameter;
+import jakarta.servlet.ServletContext;
+import jakarta.ws.rs.*;
+import jakarta.ws.rs.core.Context;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.core.Response.Status;
+import org.springframework.context.ApplicationContext;
+import org.springframework.stereotype.Component;
+import org.springframework.web.context.WebApplicationContext;
+
+import java.io.FileNotFoundException;
+
+@Component
+@Path("/hls-merger")
+public class RestService {
+
+	@Context
+	protected ServletContext servletContext;
+	Gson gson = new Gson();
+
+	
+	@POST
+	@Path("/merge/{fileName}")
```

Comment:
Please use the conventions of REST API. Here is a quick link for that
https://medium.com/@nadinCodeHat/rest-api-naming-conventions-and-best-practices-1c4e781eb6a5

Reply:
Please use the conventions for other methods as well

Reply:
Lastly, as I remember the convention in Ant Media Server starts with `rest`. I mean endpoints should be reachable through `http://{FQDN}/{APP}/rest/...`